cms/lib/studio_tabs.py

```diff
@@ -1,64 +0,0 @@
-"""Studio tab plugin manager and API."""
```

Comment:
It's a bit confusing that this got merged without being used, and is only now being removed. It would be better to only merge such things when they have some usage.